### DISCUSSION TITLE
Autoortho start, Xplane12 12.05r1 - long startup at the same airport

### DISCUSSION THREAD

**Discussion by GEC42P** [kubilus1/autoortho#291] [General]
After installation - worked well - and start the whole system (Autoortho, XP12) on Airport EGSS.
Autoortho creates the files, takes a long time but after around 20 mins the system is up and running.
Flying from EGSS to EGJB, all files are created as designed on the flight.
Starting system on the next day, same airport same route the systems takes the same time to come up.
Starting again, it takes only some minutes. I did the start and the flight 3 times.
Today I started again and .... first start takes 20 min, nothing is created
second start takes 1-2 mins
For me it looks that autoo thinks the files from yesterday are old and tries to created it again but at least it finds the files without creating anything.
System is I9-10900KF Intel CPU, 32 GB RAM, RTX 3090 24 GB, only SSD drives.

How can this be faster ?

Kind Regards Manfred

**Comment by dpdurst** [comment]
AutoOrtho is dependant on the server streaming the data, this is most likely why you see a difference in loading times.

**Reply by GEC42P** [reply]
but not if autoo doesn't load any data, and not if I start on the same day XP 5 times.
It is only the first start on the same airport on a new day.
I flew with a DA42 with 130 kts. I could see how autoo is direct infront of the building the new textures.
If I fly more than 130 kts than it starts hanging for a minute or two.

Network is not a issue with 500Mbit Up/download with fibreoptic.

Manfred

**Reply by dpdurst** [reply]
Ok, happy flying
